include/tsid/bindings/python/formulations/formulation.hpp

```diff
@@ -120,7 +120,8 @@ namespace tsid
         return self.updateRigidContactWeights(contact_name, force_regularization_weight, motion_weight);
       }
       static bool addRigidContact6dDeprecated(T& self, contacts::Contact6d & contact){
-        return self.addRigidContact(contact);
+        std::cout<<"[InverseDynamicsFormulationAccForce] Method addRigidContact(ContactBase) is deprecated. You should use addRigidContact(ContactBase, double) instead.\n";
```

Comment:
This is not the correct way for deprecated functions. You can see how we proceed with Pinocchio.